torch/csrc/jit/graph_executor_impl.h

```diff
@@ -74,7 +52,44 @@ struct GraphExecutorImplBase {
         num_outputs(this->graph->outputs().size()) {}
 
   // entry point where execution begins
-  virtual void run(Stack& stack) = 0;
+  void run(Stack& stack);
```

Comment:
i think we might need `virtual` `GraphExecutorImpl::run` eventually, but I guess, we can make it `virtual` when we need it

Reply:
It would be possible. Keep in mind that run will only get called on entry from C++ into TorchScript. Function calls go directly through the getPlan pathway. So anything that has to happen for every function call will need to be expressed as part of getPlan.